### ISSUE TITLE
Alignment not loaded error during Aligning, trimming, and inserting gap-sequences step

### ISSUE THREAD

**Issue by MiAiM** [AstrobioMike/GToTree#47]
Dear Mike, 

first of all, thanks for the great work that you’re doing! I was looking for quite some time for a tool that can handle NCBI assembly accessions. I finally found your tool that would solve my problems. 

Your documentation on Github is really excellent and I had no issues with the package installation. I eagerly wanted to test your Alteromonas example and unfortunately got stuck in the Aligning, trimming, and inserting gap-sequences' step. 

My initial command on a MacBook Pro Terminal was: 
 ```
GToTree -a alteromonas_refseq_accessions.txt -g genbank_files.txt -f fasta_files.txt -H Bacteria -t -L Species,Strain -m genome_to_id_map.tsv -j 4 -o Alteromonas_example 
 ```
The script had no problems neither with downloading all files nor with the filtering by length step. However, I encounter the following problem afterwards for every gene set as exemplified here for AICARFT_IMPCHas.

 ```
--------------------------------------------------------------------------   
	    Finished aligning and formatting gene-set AICARFT_IMPCHas.
   --------------------------------------------------------------------------   

ERROR: Alignment not loaded: "1644515152.gtotree.tmpdir/AICARFT_IMPCHas_aligned.tmp" Check the file's content.

sed: 1644515152.gtotree.tmpdir/AICARFT_IMPCHas_trimmed.faa.tmp: No such file or directory
seq: needs negative decrement

```

The script finishes and I get an output file called Alteromonas_example.tre. When I open it, there are all NCBI assembly accessions included, but with exclusively 0s as distances. 
(Our_Alteromonas_MAG:0.0,GCF_000011365.1_Bradyrhizobium_diazoefficiens_USDA110:0.0,GCF_000020585.3_Alteromonas_mediterranea_DE:0.0,GCF_000172635.2_Alteromonas_macleodii_ATCC_27126:0.0,GCF_000213655.1_Alteromonas_naphthalenivorans_SN2:0.0,GCF_000299955.1_Alteromonas_macleodii_English_Channel_673:0.0,GCF_000299995.1_Alteromonas_macleodii_Black_Sea_11:0.0,GCF_000300175.1_Alteromonas_macleodii_Balearic_Sea_AD45:0.0,GCF_000310085.1_Alteromonas_mediterranea_AltDE1:0.0,GCF_000439495.1_Alteromonas_mediterranea_Aegean_Sea_MED64:0.0,GCF_000439535.1_Alteromonas_mediterranea_Ionian_Sea_U7:0.0,GCF_000439575.1_Alteromonas_mediterranea_Ionian_Sea_UM7:0.0,GCF_000439595.1_Alteromonas_mediterranea_Ionian_Sea_UM4b:0.0,GCF_000730385.1_Alteromonas_australica_H_17:0.0,GCF_000934525.1_Alteromonas_australica_DE170:0.0,GCF_001433715.1_Alteromonas_stellipolaris_LMG_21856:0.0,GCF_001562125.1_Alteromonas_sp_Mac1_Mac1:0.0,GCF_001562195.1_Alteromonas_stellipolaris_R10SW13:0.0,GCF_001562235.1_Alteromonas_macleodii_D7:0.0,GCF_001562315.1_Alteromonas_mediterranea_UM8:0.0,GCF_001578515.1_Alteromonas_macleodii_HOT1A3:0.0,GCF_001632765.1_Alteromonas_stellipolaris_PQQ-42:0.0,GCF_001632825.1_Alteromonas_stellipolaris_PQQ-44:0.0,GCF_001886395.1_Alteromonas_mediterranea_CP48:0.0,GCF_001886415.1_Alteromonas_mediterranea_CP49:0.0,GCF_001886435.1_Alteromonas_mediterranea_RG65:0.0,GCF_001886455.1_Alteromonas_mediterranea_AR43:0.0,GCF_001886475.1_Alteromonas_sp_RW2A1_RW2A1:0.0,GCF_002831605.1_Alteromonas_sp_MB-3u-76_MB-3u-76:0.0,GCF_002849875.1_Alteromonas_macleodii_Te101:0.0,GCF_003443615.1_Alteromonas_sp_BL110_BL110:0.0);

Any help is highly appreciated.

**Comment by AstrobioMike** [comment]
Hey there, @MiAiM!

Thanks for the kind words :) 

Sorry you're having trouble, and thanks for writing in about it. 

But I can't seem to reproduce this unfortunately :(  Things work for me as expected still on my macbook pro (i tested with the latest GToTree version 1.6.12). 

I don't know if this is going to be the cause or not, but are you by chance working on one of the new macbook pros with the Apple M1 chip?

**Comment by AstrobioMike** [comment]
scratch that, @MiAiM! @jianshu93 just helped us pin point exactly what is going on here by notifying me there is a pretty bit new muscle release https://github.com/AstrobioMike/GToTree/issues/48 :) 

If doing a conda install within the past 15 hours, it would have grabbed the latest muscle version, which indeed leads to this error.

I'm fixing ASAP and will update then

Thank you both!

**Comment by AstrobioMike** [comment]
This should be solved as of version 1.6.20, @MiAiM  👍 

It was a consequence of how I was (intentionally) loose with letting conda choose the muscle version, and muscle version 5 becoming available on conda today (which couldn't be run in the same fashion as the earlier versions, leading to the alignment files missing). 

Thanks again for writing in about it so I could address this so quickly!

**Comment by MiAiM** [comment]
How fortunate that @jianshu93 posted his comment. Thanks a bunch @AstrobioMike for the great support and the fix!
